pkgs/applications/audio/openmpt123/default.nix

```diff
@@ -1,24 +1,30 @@
-{ stdenv, fetchurl, SDL2, pkgconfig, flac, libsndfile }:
+{ stdenv, fetchurl, zlib, pkgconfig, mpg123, libogg, libvorbis, portaudio, libsndfile, flac }:
 
 let
-  version = "0.2.7025-beta20.1";
+  version = "0.4.1";
 in stdenv.mkDerivation rec {
   name = "openmpt123-${version}";
+
   src = fetchurl {
-    url = "https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${version}.tar.gz";
-    sha256 = "0qp2nnz6pnl1d7yv9hcjyim7q6yax5881k1jxm8jfgjqagmz5k6p";
+    url = "https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${version}+release.autotools.tar.gz";
+    sha256 = "1k1m1adjh4s2q9lxgkf836k5243akxrzq1hsdjhrkg4idd3pxzp4";
   };
+
+  enableParallelBuilding = true;
+  doCheck = true;
+
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ SDL2 flac libsndfile ];
-  makeFlags = [ "NO_PULSEAUDIO=1 NO_LTDL=1 TEST=0 EXAMPLES=0" ]
-  ++ stdenv.lib.optional (stdenv.isDarwin) "SHARED_SONAME=0";
-  installFlags = "PREFIX=\${out}";
+  buildInputs = [ zlib mpg123 libogg libvorbis portaudio libsndfile flac ];
+
+  configureFlags = [
```

Comment:
Note we often adjust this flag using `usePulseAudio = config.pulseaudio or stdenv.isLinux;`

Reply:
@FRidh Done, thanks!